chore: update stale issue workflow

peterwoodworth opened the pull request:
*Description of changes:* redo of https://github.com/aws/aws-sdk/pull/400 but with signed commits, and with updated ancient time

closes #400, closes #462

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

aBurmeseDev reviewed (commented):
Added single comment on stale-issue-cleanup version.

peterwoodworth commented on .github/workflows/close-stale-issues.yml:
```diff
@@ -1,30 +1,51 @@
     name: Stale issue job
     steps:
-    - uses: aws-actions/stale-issue-cleanup@v3
+    - uses: aws-actions/stale-issue-cleanup@v5
```
Done

peterwoodworth commented:
New changes in commit: Removed the ancient message prop, removed feature-request from exempt labels (not necessary with ancient message disabled), and bumped to v6

aBurmeseDev reviewed (approved):
🚀
